docs/UserGuide.adoc

```diff
@@ -46,100 +47,210 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 ====
 *Command Format*
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-* Items in square brackets are optional e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-* Items with `…`​ after them can be used multiple times including zero times e.g. `[t/TAG]...` can be used as `{nbsp}` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
-* Parameters can be in any order e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+* Words in bold are the parameters to be supplied by the user +
+e.g. in `add **n/NAME**`, `**NAME**` is a parameter which can be used as `add n/John Doe`.
+* Items in square brackets represent the possible options to specify e.g in this command format: +
+`add [patient/donor/doctor] n/**NAME**`, this means we can choose patient, donor, or doctor.
+* `[attributes]` refer to all the attributes in the <<List of Attributes>>.
+* Parameters can be in any order e.g. if the command specifies `n/**NAME** c/**CONTACT**`, `c/**CONTACT** n/**NAME**` +
+is also acceptable.
```

Comment:
can change c/ to p/ because they gave p/ as the code base and it will need extra work if we change it to c/ and this change doesnt add any value to the project.

Reply:
Rectified this in my latest commit